hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/ClusteringPlanActionExecutor.java

```diff
@@ -102,6 +104,16 @@ public Option<HoodieClusteringPlan> execute() {
         throw new HoodieIOException("Exception scheduling clustering", ioe);
       }
     }
+
+    if (config.isTableManagerSupportAction(ActionType.replacecommit)) {
+      createClusteringToService();
```

Comment:
rename all such createXXX to delegateXXX to clearly capture what's going on.

Reply:
will update.